packages/config/config/devices/0x0438/touch-thermostat-16a.json

```diff
@@ -0,0 +1,348 @@
+			"description": "Set the mode after drying mode completes.",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 3,
+			"defaultValue": 2,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Off mode",
+					"value": 0
+				},
+				{
+					"label": "Manual mode",
+					"value": 1
+				},
+				{
+					"label": "Energy save mode",
+					"value": 2
+				},
+				{
+					"label": "Away mode",
+					"value": 3
+				}
+			]
+		},
+
+		{
+			"#": "16",
+			"label": "Temperature Report on Value",
+			"description": "When to report temperature relative change actively. Allowable range: 0 (disable), 2-10",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 10,
+			"unit": "0.1 °C",
+			"defaultValue": 2
+		},
+
+		{
+			"#": "17",
+			"label": "Temperature Report on Interval",
+			"description": "Time interval to report temperature. Allowable range: 0 (disable), 30-28800",
+			"valueSize": 2,
+			"minValue": 0,
+			"maxValue": 28800,
+			"unit": "seconds",
+			"defaultValue": 300
+		},
+
+		{
+			"#": "18",
+			"label": "Power Change Threshold",
+			"description": "Threshold when to report power change. Allowable range: 0 (disable), 1-100",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 100,
+			"unit": "W",
+			"defaultValue": 5
+		},
+
+		{
+			"#": "19",
+			"label": "Current Change Threshold",
+			"description": "Threshold when to report current change. Allowable range: 0 (disable), 1-10",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 10,
+			"unit": "0.1 A",
+			"defaultValue": 10
+		},
+
+		{
+			"#": "20",
+			"label": "Voltage Change Threshold",
+			"description": "Threshold when to report voltage change. Allowable range: 0 (disable), 1-10",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 10,
+			"unit": "V",
+			"defaultValue": 2
+		},
+
+		{
+			"#": "21",
+			"label": "Meter Report Interval",
+			"description": "Time interval for active report of Meter. Allowable range: 60-2678400",
+			"valueSize": 4,
+			"minValue": 60,
+			"maxValue": 2678400,
+			"unit": "seconds",
+			"defaultValue": 600
+		},
+
+		{
+			"#": "22",
+			"label": "OLED Brightness",
+			"description": "Sets OLED brightness, only when active.",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 2,
+			"defaultValue": 1,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Low",
+					"value": 0
+				},
+				{
+					"label": "Medium",
+					"value": 1
+				},
+				{
+					"label": "High",
+					"value": 2
+				}
+			]
+		},
+		{
+			"#": "23",
+			"label": "Sensor to Display",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Air",
+					"value": 0
+				},
+				{
+					"label": "Floor",
+					"value": 1
+				}
+			]
+		},
+
+		{
+			"#": "24",
+			"label": "Timezone",
+			"description": "The timezone that the parameters command adopts is UTC, the timezone need to be set. Allowable range: -12 - 12",
```

Comment:
No need to restate the range if it matches min/maxValue.
This sentence also seems to be broken - I'm not sure what its supposed to say.

Maybe the label should just be changed to `UTC Timezone Offset` with a `unit` of `hours`.